contracts/smart-contract-wallet/modules/SessionValidationModules/ISessionValidationModule.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+import {UserOperation} from "../BaseAuthorizationModule.sol";
+
+interface ISessionValidationModule {
+    function validateSessionUserOp(
+        UserOperation calldata _op,
+        bytes32 _userOpHash,
+        bytes calldata _data,
```

Comment:
can we rename these vars to more readable values?
also if you can check missing natspecs for all new contracts and functions

Reply:
yes! already todo-ed adding natspec. will rename as well

Reply:
done